lib/publiccloud/instance.pm

```diff
@@ -336,6 +336,8 @@ sub wait_for_guestregister {
     wait_for_ssh([timeout => 600] [, proceed_on_failure => 0])
 
 Check if the SSH port of the instance is reachable and open.
+Also check system UP
+
 =cut
 
 sub wait_for_ssh {
```

Comment:
I would suggest to rename a function so it will be more accurate for what is going on below . we not just "wait for ssh" anymore ...

Reply:
But renaming this function it'd mean also to change name in many many modules using it now: should we do it the same?

Reply:
ok :) then changing there the name too will help me to take the count of updates

Reply:
ok